lib/commands/doctor/checks/mysql.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const execa = require('execa');
+const chalk = require('chalk');
+const includes = require('lodash/includes');
+
+const errors = require('../../../errors');
+
+function mysqlCheck(ctx) {
+    // On ubuntu, mysqld is in `/usr/sbin` but it's not automatically in the PATH of non-root users
+    // So, we modify the path env var to make things work
+    const options = ctx.system.platform.linux ? {env: {PATH: `/usr/sbin:${process.env.PATH}`}} : {};
+
+    // Technically this doesn't work on windows, but there's
+    // not an easy way to do that anyways so ¯\_(ツ)_/¯
+    return execa.shell('which mysqld', options).catch(() => {
+        ctx.ui.log(`${chalk.yellow(`Local MySQL install not found. You can ignore this if you are using a remote MySQL host.
+Alternatively you could:`)}
+${chalk.blue('a)')} install MySQL locally
+${chalk.blue('b)')} run ${chalk.cyan('`ghost install --db=sqlite3`')} to use sqlite
```

Comment:
This clashes with the [docs](https://docs.ghost.org/docs/cli-install) wrt the `dbpath` flag

Reply:
The docs are incorrect (I think) there - you don't have to pass the dbpath flag.